src/components/molecules/Inbox.vue

```diff
@@ -2,27 +2,30 @@
   <h1 class="font-heading font-bold text-4xl pl-3 pt-4">Inbox</h1>
   <div class="md:overflow-auto space-y-1">
     <inbox-item
+      v-for="(inboxItem, index) in inboxItems"
+      :key="index"
```

Comment:
Instead of the index, we could use the inboxItem.id for the key. What do you think?

P.S My comment might be completely useless x)

Reply:
It makes more sense to me. I updated it but idk if it will ever get used